### ISSUE TITLE
Accelerometer calibrating problem 

### ISSUE THREAD

**Issue by tgreif** [iNavFlight/inav#2781]
Sp Racing F3 "LHI Official Store"
INAV 1.8.0 /Chrome: 61.0.3163.100, Configurator: 1.8.0

### Behavior
hi everybody 
i've installed Inav to my new SP racing F3 controller, with external OLED display Barometer (i2C)
magnetometer (i2C) and gps (Seriale) everything is working well . but i'm not able to calibrate my Accelerometer (sure i'm using the 6 point calibration) . 
I've tryed so many times in different conditions. 

Please give me a hint where I can make it work 
Is this a new feature request?_

**Comment by shellixyz** [comment]
Make sure you calibrate the accelerometer relative to the board not relative to the model. The arrow in [accelerometer-calibration-steps](https://github.com/iNavFlight/inav/wiki/Sensor-calibration#accelerometer-calibration-steps) represents the arrow on the board not the front of the model.

**Comment by tgreif** [comment]
hi shellixyz. thanks for response but i think that i did like this . 
![image](https://user-images.githubusercontent.com/2463499/36279782-84ed626e-1298-11e8-840c-96a09ee8fb7e.png)
where yhe arrow is the direction of arrow on FC. 
"top,bottom,left up right down.

**Comment by giacomo892** [comment]
As soon you did 1 and 2 you can randomly do the other 4 steps and it should work ok.

**Comment by tgreif** [comment]
yes it should but  unfurtunatly it does not, 
thanks for response . 
i'm hoping in the new version if inav  there is an tool.

**Comment by shellixyz** [comment]
What do you mean by "the calibration doesn't work" ? You get a red cross in the configurator next to "Accelerometer is calibrated" ? In the next version of the configurator I think there will be a guide to indicate each step but the procedure will stay the same. After the calibration what are the values of the `acczero_*` and `accgain_*` settings ?

**Comment by tgreif** [comment]
hi , 
i mean the red cross  in the configurator,, 
![image](https://user-images.githubusercontent.com/2463499/36282497-a3f7ea9a-12a1-11e8-964e-caa4848a0e01.png)
accgain 4096-
![image](https://user-images.githubusercontent.com/2463499/36282672-33c111c4-12a2-11e8-8673-3b41b2850f6a.png)
instead acczero is 32760
![image](https://user-images.githubusercontent.com/2463499/36282725-609f895a-12a2-11e8-9d73-b8f1fc9afcf0.png)

**Comment by tgreif** [comment]
the  cleanflight and betafliegt (simple calibration work)

**Comment by tgreif** [comment]
i send only the first fotos , top and bottom for limit the data 
![img_20180215_225254](https://user-images.githubusercontent.com/2463499/36283361-9ae483de-12a4-11e8-859c-fad3c20ac7d0.jpg)
![img_20180215_225300](https://user-images.githubusercontent.com/2463499/36283363-9b04a0ce-12a4-11e8-9292-eadbbef79c37.jpg)

**Comment by shellixyz** [comment]
You can check if your accelerometer is working correctly. In the configurator go in the sensors tab and look at the accelerometer curves. When your FC is flat the green and blue lines should be near 0. When on the side the green and red lines should be near 0. And when on the front the blue and red lines should be 0.

**Comment by tgreif** [comment]
thanks . 
i think there is an problem on it 
![image](https://user-images.githubusercontent.com/2463499/36285361-89a22bce-12ab-11e8-9042-b8ef0339c9cf.png)
blue line is always under 0 between -3 and -4 depending on the position. 
so i will try to change it , i have an external board with the 6050 . i will connect to i2c . 
do i need to cut the original one ?? what pin?

**Comment by shellixyz** [comment]
Indeed that would explain why the 3 axes calibration does not work. You probably can use an external board with a 6050 MPU but not without modifying the source code I'm afraid. The code only expects the MPU to be connected to the "internal" I2C bus. If you have the tools and the skills to do it the best would be to replace the defective MPU.

**Comment by tgreif** [comment]
thank , i think that i will do the next days in my lab. 
WHAT DO YOY THING WHAT IS THE BEST FC for inav ?

**Comment by shellixyz** [comment]
The best one is the one suited for your project. It depends what you want to do with it. What sensors you need, how many UARTs, how many outputs, the maximum size... I personally often use Omnibus boards with F3 or F4 MCUs.

**Comment by junwoo091400** [comment]
Just Maybe related to this??

https://github.com/iNavFlight/inav/issues/2806

**Comment by shellixyz** [comment]
I was asking myself the same thing but I'm not sure the PR for the fix changes the way the accelerometer data is read. His issue was clearly that the data for the X axis was wrong.

**Comment by tgreif** [comment]
hi everybody . thanks a lot to all. 
i 've tried a cc3d and everything is working well. 
today is arrived my new sp racing f3 . only 3 minutes to set up and calibrate. 
i will change or disable the  broken Accelerometer on the other fc . 
now i will first do everything with my new one . .
today i try to connect my Bluetooth on the serial for to connect with the android device . 
bye

**Comment by DavidFpv** [comment]
hello,
I'm trying to calibrate my flight controller and ran into a problem.
When I point it to the right the calibration is done as if the board is pointing down.

![image](https://user-images.githubusercontent.com/56543071/96331493-5c871f80-1066-11eb-9be7-7a69a83aa746.png)

**Comment by Sedgee** [comment]
> 
> 
> hello,
> I'm trying to calibrate my flight controller and ran into a problem.
> When I point it to the right the calibration is done as if the board is pointing down.
> 
> ![image](https://user-images.githubusercontent.com/56543071/96331493-5c871f80-1066-11eb-9be7-7a69a83aa746.png)

Hi David,
Did you manage to resolve this?
I'm getting the exact same problem with a GEPRC SPAN F722 HD BT FC board using INAV 2.6. The board is aligned correctly with the arrow on top pointing forward.
I have corrected 90deg yaw which makes the graphic in Setup page correct but when I go into sensors it shows that the orientation is out by 90deg.
If I set yaw back to 0deg then the Sensors page shows it working correctly but the Setup page graphic is out 90deg on yaw.
Any help would be appreciated.
